include/psa/crypto_values.h

```diff
@@ -634,7 +651,8 @@
  *         algorithm identifier.
  */
 #define PSA_ALG_IS_HASH(alg)                                            \
-    (((alg) & PSA_ALG_CATEGORY_MASK) == PSA_ALG_CATEGORY_HASH)
+    ((((alg) & PSA_ALG_CATEGORY_MASK) == PSA_ALG_CATEGORY_HASH) &&      \
+      !PSA_ALG_IS_POLICY(alg))
```

Comment:
Minor: if `PSA_ALG_POLICY_FLAG(PSA_ALG_ANY_HASH)` remains false, this could still be written as `(alg & MASK) == CATEGORY_xxx` where `MASK` includes the policy flag bit.

Reply:
N/A now that the policy bit is removed.